vars/checkmarxExecuteScan.groovy

```diff
@@ -39,7 +39,7 @@ void call(Map parameters = [:]) {
                 passwordVariable: 'password',
                 usernameVariable: 'username'
```

Comment:
change to `PIPER_username` then you don't need to pass it as parameter any longer :-)